src/main/java/mops/security/exception/WriteAccessPermission.java

```diff
@@ -0,0 +1,12 @@
+package mops.security.exception;
+
+public class WriteAccessPermission extends Exception {
```

Comment:
can extend the new `MopsException`

Reply:
will do and will rename with exception at the end, was waiting to do after you guys finished #105